docs/EiffelVis/content/_index/features-2/timeline.md

```diff
@@ -0,0 +1,17 @@
++++
+fragment = "item"
+#disabled = false
+date = "2022-01-05"
+weight = 115
+background = "dark"
+align = "top"
+
+title = "Timeline"
```

Comment:
This feature is bad and will probably go at some point, lets not advertise it for now

Reply:
Ok will do